user_manual/docs/controller.md

```diff
@@ -0,0 +1,12 @@
+DMX / RDM Controller {#controller}
+=======================
+
+[TOC]
+
+# Overview {#controller-overview}
+
+The device will automatically switch to controller mode when it's configured as
+an output port within OLA.
+
+When operating in controller mode, remember to set the biasing switches
+according to the manufacturer's instructions.
```

Comment:
I know you've discussed this elsewhere, I think we should actually specify what biasing is required, and say see the manufacturer instructions for how to use it. As the termination should be the same regardless of who made it, it's just how to do it that changes.